lighthouse-core/audits/byte-efficiency/duplicated-javascript.js

```diff
@@ -164,9 +163,7 @@ class DuplicatedJavascript extends ByteEfficiencyAudit {
         /** @type {number|undefined} */
         let transferRatio = transferRatioByUrl.get(url);
         if (transferRatio === undefined) {
-          const networkRecord = url === artifacts.URL.finalUrl ?
-            mainDocumentRecord :
-            networkRecords.find(n => n.url === url);
+          const networkRecord = getRequestForScript(networkRecords, script);
```

Comment:
could `const url` be removed?

Reply:
It's still used in multiple places, I think we can leave it.